### PR TITLE
Bump min SDK version to non-prerelease version

### PR DESCRIPTION
**Pull request by gspencergoog**
The old min SDK constraint was a pre-release version, which causes a warning when publishing.  This just removes the '-0' and bumps it to the next version (2.14.0).

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ description: A pluggable, mockable process invocation abstraction for Dart.
 homepage: https://github.com/google/process.dart
 
 environment:
-  sdk: '>=2.14.0-0 <4.0.0'
+  sdk: '>=2.15.0 <4.0.0'
```
Comment (zanderso) [inline pubspec.yaml]: Does this change need a package version bump?

Comment (gspencergoog) [reply pubspec.yaml]: 5.0.0 hasn't been published yet, so I don't think so.

Comment (jakemac53) [reply pubspec.yaml]: A more direct translation would be using `2.14.0`, did we need to bump it to 2.15?

Comment (gspencergoog) [reply pubspec.yaml]: Yes, good point.  I'll move it back to 2.14.0.

**Review by jakemac53 — APPROVED** [review]

**Comment by gspencergoog** [comment]
published to pub

**Pull request merged**